pom.xml

```diff
@@ -139,6 +139,7 @@
     <version.org.apache.httpcomponents.httpclient>4.5.4</version.org.apache.httpcomponents.httpclient>
     <version.org.apache.httpcomponents.httpcore>4.4.6</version.org.apache.httpcomponents.httpcore>
     <version.org.apache.karaf>2.4.0</version.org.apache.karaf>
+    <version.org.apache.kafka>2.6.0</version.org.apache.kafka>
```

Comment:
Not pretty sure about this, but what about Red Hat version for this artifact?

https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients/2.3.0.redhat-00001

Reply:
productization artifacts are only by prod versions.